core/controllers/profile.py

```diff
@@ -374,27 +374,33 @@ class UserInfoHandler(base.BaseHandler):
 
```

Comment:
In the docstring above, explain what happens if there is no currently logged-in user.

Reply:
Okay

Reply:
Done

Reply:
Done